CreeDictionary/API/models.py

```diff
@@ -332,7 +333,9 @@ def save(self, *args, **kwargs):
         super(Wordform, self).save(*args, **kwargs)
 
     @staticmethod
-    def fetch_lemma_by_user_query(user_query: str, **kwargs) -> "CreeAndEnglish":
+    def fetch_lemma_by_user_query(
+        user_query: str, **extra_constraints
+    ) -> "CreeAndEnglish":
```

Comment:
renamed `kwargs` to `extra_constraints`. I think it's more descriptive, as the kwargs are exclusively used for that.